lib/cluster.js

```diff
@@ -285,6 +287,8 @@ function masterInit() {
     var debugPort = process.debugPort + id;
     var hasDebugArg = false;
 
+    debugPort = debugPort % kDebugPortRange + kDebugPortStart;
```

Comment:
debugPort is equals to 12345 only in cluster's master. First worker created will receive 12346. After my fix it'll receive 12346 % kDebugPortRange + 1024. Yes, behaviour is changed, but I'm wondering was it documented? One have no any control on worker's debugPort value, isn't it?